diff -ruN original/microchip_solutions_v2013-06-15/TCPIP/Demo_App/applic.make modified/microchip_solutions_v2013-06-15/TCPIP/Demo_App/applic.make
--- original/microchip_solutions_v2013-06-15/TCPIP/Demo_App/applic.make	1970-01-01 01:00:00.000000000 +0100
+++ modified/microchip_solutions_v2013-06-15/TCPIP/Demo_App/applic.make	2017-11-04 02:16:05.199978922 +0100
@@ -0,0 +1,60 @@
+MCU      = 18F97J60
+CFLAGS     = --extended -mL -k -Ou- -Ot- -Ob- -Op- -Or- -Od- -Opa-
+INCLUDE  = -I ../../Microchip/Include
+MACROS     += -DCFG_INCLUDE_PICDN2_ETH97
+LDMACROS   += -u_EXTENDEDMODE
+MACROS_GCC += -D__EXTENDED18__
+
+CSRC = \
+	../../Microchip/TCPIP_Stack/ARP.c \
+	../../Microchip/TCPIP_Stack/Delay.c \
+	../../Microchip/TCPIP_Stack/DNS.c \
+	../../Microchip/TCPIP_Stack/ETH97J60.c \
+	../../Microchip/TCPIP_Stack/Hashes.c \
+	../../Microchip/TCPIP_Stack/Helpers.c \
+	../../Microchip/TCPIP_Stack/HTTP2.c \
+	../../Microchip/TCPIP_Stack/ICMP.c \
+	../../Microchip/TCPIP_Stack/IP.c \
+	../../Microchip/TCPIP_Stack/LCDBlocking.c \
+	../../Microchip/TCPIP_Stack/MPFS2.c \
+	../../Microchip/TCPIP_Stack/SPIEEPROM.c \
+	../../Microchip/TCPIP_Stack/StackTsk.c \
+	../../Microchip/TCPIP_Stack/TCP.c \
+	../../Microchip/TCPIP_Stack/Tick.c \
+	../../Microchip/TCPIP_Stack/UDP.c \
+	../../Microchip/TCPIP_Stack/SMTP.c \
+	CustomHTTPApp.c \
+	MainDemo.c \
+	PingDemo.c \
+	SMTPDemo.c \
+#\
+	CustomSNMPApp.c \
+	UARTConfig.c \
+	../../Microchip/TCPIP_Stack/Announce.c \
+	../../Microchip/TCPIP_Stack/AutoIP.c \
+	../../Microchip/TCPIP_Stack/TFTPc.c \
+	../../Microchip/TCPIP_Stack/DHCP.c \
+	../../Microchip/TCPIP_Stack/DHCPs.c \
+	../../Microchip/TCPIP_Stack/DNSs.c \
+	../../Microchip/TCPIP_Stack/ENC28J60.c \
+	../../Microchip/TCPIP_Stack/ENCX24J600.c \
+	../../Microchip/TCPIP_Stack/UART.c \
+	../../Microchip/TCPIP_Stack/UART2TCPBridge.c \
+	../../Microchip/TCPIP_Stack/UDPPerformanceTest.c \
+	../../Microchip/TCPIP_Stack/TCPPerformanceTest.c \
+	../../Microchip/TCPIP_Stack/Telnet.c \
+	../../Microchip/TCPIP_Stack/FTP.c \
+	../../Microchip/TCPIP_Stack/SPIFlash.c \
+	../../Microchip/TCPIP_Stack/SPIRAM.c \
+	../../Microchip/TCPIP_Stack/SNTP.c \
+	../../Microchip/TCPIP_Stack/SNMP.c \
+	../../Microchip/TCPIP_Stack/DynDNS.c \
+	../../Microchip/TCPIP_Stack/NBNS.c \
+	../../Microchip/TCPIP_Stack/Reboot.c \
+	../../Microchip/TCPIP_Stack/BerkeleyAPI.c \
+	BerkeleyTCPClientDemo.c \
+	BerkeleyTCPServerDemo.c \
+	BerkeleyUDPClientDemo.c \
+	GenericTCPClient.c \
+	GenericTCPServer.c \
+
diff -ruN original/microchip_solutions_v2013-06-15/TCPIP/Demo_App/Configs/TCPIP_ETH97.h modified/microchip_solutions_v2013-06-15/TCPIP/Demo_App/Configs/TCPIP_ETH97.h
--- original/microchip_solutions_v2013-06-15/TCPIP/Demo_App/Configs/TCPIP_ETH97.h	2017-11-04 01:27:21.414056742 +0100
+++ modified/microchip_solutions_v2013-06-15/TCPIP/Demo_App/Configs/TCPIP_ETH97.h	2017-11-04 01:48:36.036883100 +0100
@@ -61,8 +61,8 @@
  *   Uncomment or comment the following lines to enable or
  *   disabled the following high-level application modules.
  */
-#  define STACK_USE_UART					// Application demo using UART for IP address display and stack configuration
-#  define STACK_USE_UART2TCP_BRIDGE			// UART to TCP Bridge application example
+// #  define STACK_USE_UART					// Application demo using UART for IP address display and stack configuration
+// #  define STACK_USE_UART2TCP_BRIDGE			// UART to TCP Bridge application example
 //#define STACK_USE_IP_GLEANING
 #  define STACK_USE_ICMP_SERVER				// Ping query and response capability
 #  define STACK_USE_ICMP_CLIENT				// Ping transmission capability
@@ -157,24 +157,24 @@
 #  define MY_DEFAULT_MAC_BYTE5            (0x00)// internal factory programmed MAC
 #  define MY_DEFAULT_MAC_BYTE6            (0x00)// address instead.
 
-#  define MY_DEFAULT_IP_ADDR_BYTE1        (169ul)
-#  define MY_DEFAULT_IP_ADDR_BYTE2        (254ul)
-#  define MY_DEFAULT_IP_ADDR_BYTE3        (1ul)
-#  define MY_DEFAULT_IP_ADDR_BYTE4        (1ul)
+#  define MY_DEFAULT_IP_ADDR_BYTE1        (192ul)
+#  define MY_DEFAULT_IP_ADDR_BYTE2        (168ul)
+#  define MY_DEFAULT_IP_ADDR_BYTE3        (0ul)
+#  define MY_DEFAULT_IP_ADDR_BYTE4        (254ul)
 
 #  define MY_DEFAULT_MASK_BYTE1           (255ul)
 #  define MY_DEFAULT_MASK_BYTE2           (255ul)
-#  define MY_DEFAULT_MASK_BYTE3           (0ul)
+#  define MY_DEFAULT_MASK_BYTE3           (255ul)
 #  define MY_DEFAULT_MASK_BYTE4           (0ul)
 
-#  define MY_DEFAULT_GATE_BYTE1           (169ul)
-#  define MY_DEFAULT_GATE_BYTE2           (254ul)
-#  define MY_DEFAULT_GATE_BYTE3           (1ul)
+#  define MY_DEFAULT_GATE_BYTE1           (192ul)
+#  define MY_DEFAULT_GATE_BYTE2           (168ul)
+#  define MY_DEFAULT_GATE_BYTE3           (0ul)
 #  define MY_DEFAULT_GATE_BYTE4           (1ul)
 
-#  define MY_DEFAULT_PRIMARY_DNS_BYTE1	(169ul)
-#  define MY_DEFAULT_PRIMARY_DNS_BYTE2	(254ul)
-#  define MY_DEFAULT_PRIMARY_DNS_BYTE3	(1ul)
+#  define MY_DEFAULT_PRIMARY_DNS_BYTE1	(192ul)
+#  define MY_DEFAULT_PRIMARY_DNS_BYTE2	(168ul)
+#  define MY_DEFAULT_PRIMARY_DNS_BYTE3	(0ul)
 #  define MY_DEFAULT_PRIMARY_DNS_BYTE4	(1ul)
 
 #  define MY_DEFAULT_SECONDARY_DNS_BYTE1	(0ul)
diff -ruN original/microchip_solutions_v2013-06-15/USB/Device_-_HID_-_Mouse/Firmware/applic.make modified/microchip_solutions_v2013-06-15/USB/Device_-_HID_-_Mouse/Firmware/applic.make
--- original/microchip_solutions_v2013-06-15/USB/Device_-_HID_-_Mouse/Firmware/applic.make	1970-01-01 01:00:00.000000000 +0100
+++ modified/microchip_solutions_v2013-06-15/USB/Device_-_HID_-_Mouse/Firmware/applic.make	2017-11-04 01:54:14.979321152 +0100
@@ -0,0 +1,10 @@
+MCU      = 18F97J94
+LKR      = rm18f97j94_g.lkr
+CFLAGS   = -Ou- -Ot- -Ob- -Op- -Or- -Od- -Opa-
+
+CSRC = \
+       usb_descriptors.c \
+       mouse.c \
+       ../../../Microchip/USB/usb_device.c \
+       ../../../Microchip/USB/HID_Device_Driver/usb_function_hid.c \
+
